### PR TITLE
User feedback

### LINKED ISSUE #104: User's feedback

### PR DESCRIPTION
**Pull request by Atos20**
### What does this PR do?
- prevents the user from typing letters into the input field
- moves the leaves from the report to make the report more readable 
-makes sure all test pass after the changes
### Where should the reviewer start?
- `src/component/Question/Question/tsx`
### How should this be manually tested?
run `npm test ` and `npm start`
### Any background context you want to provide?
- made changes based on user's feedback
### What are the relevant tickets?
closes #104

### CODE REVIEW

File: src/components/Question/Question.tsx [hunk 1]
```diff
@@ -96,12 +97,11 @@ export const Question: React.FC<Props> = ({updateAllAnswers}) => {
 
         <div className="input-box">
           <input
-            placeholder='amount'
+            placeholder='your answer'
```
Comment (Arique1104) [inline src/components/Question/Question.tsx:100]: Yay!  Look at that!

Comment (Arique1104) [reply src/components/Question/Question.tsx:100]: Thank you for making that change.

**Review by Arique1104 — APPROVED** [review]
This looks so cool!  Thank you for doing this!

**Review by BlakeDonn — APPROVED** [review]
thanks for adding these changes in, definitely improves the UX

**Pull request merged**